test/src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -911,6 +923,117 @@ public DumbSlave createSlave(Label l, EnvVars env) throws Exception {
     }
 
     /**
+     * Get JSON from A Jenkins endpoint.
+     * @param url The endpoint URL.
+     * @return The JSON.
+     */
+    public JSONWebResponse getJSON(@Nonnull String url) throws IOException, SAXException {
+        JenkinsRule.WebClient webClient = createWebClient();
+
+        if (url.startsWith("/jenkins/")) {
+            url = url.substring("/jenkins/".length());
+        } else if (url.startsWith("/")) {
+            url = url.substring(1);
```

Comment:
Correct, the caller is responsible for specifying a `url` relative to the context path.